### PR TITLE
implement accessors and serializaion for pomodoro duration

### PR DESCRIPTION
**Pull request by alx-ef**

### CODE REVIEW

File: internal/userconfig/userconfig.go [hunk 1]
```diff
@@ -110,3 +112,34 @@ func mapConfigButtonNamesToRealNames(configNames []string) []string {
 
 	return realNames
 }
+
+type strDuration time.Duration
+
+func (d *strDuration) UnmarshalJSON(b []byte) error {
```
Comment (zakirullin) [inline internal/userconfig/userconfig.go]: I would suggest to simply everything.

I believe there's no practical reason to set pomodoro time in any units but minutes.

We can just save/load one number like 20 or 50, and that would be minutes. I doubt someone would need seconds. 

And If we need seconds for testing purposes - we can use zero, for example 🤔

Comment (alx-ef) [reply internal/userconfig/userconfig.go]: I do use 1 second in test. And if i replace it with 0, it would't become a corner case and i will not be able to test the whole happy pass. 
Instead, i can store this value as string and convert to duration in getter.
